iml-api/src/graphql/mod.rs

```diff
@@ -485,6 +464,99 @@ impl QueryRoot {
 
         Ok(xs)
     }
+
+    #[graphql(arguments(
+        limit(description = "optional paging limit, defaults to all rows"),
```

Comment:
I don't think we should default to all rows for this; there could potentially be millions of rows.

Perhaps 100 is a reasonable choice.

Reply:
Done